quotes/static/quotes/css/quotes.css

```diff
@@ -0,0 +1,84 @@
+
+.profilepage__thumbwrapper {
+    margin-top: 3em;
+}
+
+.profilepage__position {
+    margin-top: 2em;
+    margin-bottom: 0.25em;
+}
+
+.profilepage__ksginfo > p {
+    margin: 0.3em 0;
+}
+
+.quote__text {
+    font-style: italic;
+}
+
+.no-bullet {
+    list-style-type: none;
+}
+
+.quote_box {
+
+}
+
+.grid-container {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    padding: 10px;
+
+}
+
+.grid-item {
+    background-color: rgba(255, 255, 255, 0.8);
+    padding: 10px;
+    font-size: 14px;
+    text-align: left;
+    font-style: italic;
+    width: 200px;
+    margin: 10px;
+}
+
+a:link {
```

Comment:
All these link classes pollute the "global" stylings by resetting all links.
Have a look at the class `.resetlink` in main.css, it probably does exactly what you want. (It inherits the color, meaning it typically reverts to being black)